_component_design/tabset.md

```diff
@@ -137,14 +218,15 @@ Hovering over another tab changes the color of the text. Hovering off the tab ch
 
 ### Pressed state
 
-Clicking the text on the tab changes the color by a few shades to visually confirm an action.
+To visually confirm to a user that a tab is selected, change the color of the tab text by a few shades.
```

Comment:
To visually reinforce the tab selection, change the color of the tab text to the pressed state for the duration of the click. Change the tab text to selected state after the click.